### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by Nany-fork**
Hasta este punto esta instalada y configurada la gema de devise, mi base de datos , el modelo de usuario una landing.

### CODE REVIEW

File: db/migrate/20220127021102_devise_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      t.string :name
+      t.string :lastName
```
Comment (itsmeurbi) [inline db/migrate/20220127021102_devise_create_users.rb:36]: Still not being came_case

File: db/migrate/20220128041100_rename_last_name_tolast_name.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RenameLastNameTolastName < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :users, :lastName, :last_name
```
Comment (itsmeurbi) [inline db/migrate/20220128041100_rename_last_name_tolast_name.rb:3]: Same here

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+    protect_from_forgery with: :exception
```
Comment (itsmeurbi) [inline app/controllers/application_controller.rb:2]: There a mixture between 2 and 4 sauces for a tab.?
I strongly recommend you to use 2 spaces. If you rather want to use 4 it's ok, but let's just keep consistency

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
Still having a couple of things to do. They really minor changes, don't hesitate

**Comment by itsmeurbi** [comment]
I just realize you have a `develop` branch, which already includes `users`
This PR it's pointing to `main`, so now I wonder the reason behind that 🤔

File: db/migrate/20220128041100_rename_last_name_tolast_name.rb (hunk 2 above)
Comment (Nany-fork) [reply db/migrate/20220128041100_rename_last_name_tolast_name.rb:3]: its the migration to rename urbi in the schema its fixed

Comment (itsmeurbi) [reply db/migrate/20220128041100_rename_last_name_tolast_name.rb:3]: My bad. But there's no need to create an extra migration. Why don't we change the [previous](https://github.com/Nany-fork/ledger_account/pull/2/files#diff-1252a1438465bbe534cd562038a1c4d550904ebc6f9f7e1e1f7ca28e470e4a3dR1) one instead? I meant it's not in prod yet

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit!
Well done!

**Pull request merged**